### PR TITLE
New component tplink_lte with SMS notify service

### PR DESCRIPTION
**Pull request by andtos90**
**Description:**

**Pull request in [home-assistant](https://github.com/home-assistant/home-assistant) (if applicable):** home-assistant/home-assistant#17111

## Checklist:

- [X] Branch: `next` is for changes and new documentation that will go public with the next [home-assistant](https://github.com/home-assistant/home-assistant) release. Fixes, changes and adjustments for the current release should be created against `current`.
- [X] The documentation follow the [standards][standards].

[standards]: https://home-assistant.io/developers/documentation/standards/

### CODE REVIEW

**Review by frenck — DISMISSED** [review]

**Comment by frenck** [comment]
Thanks, @andtos90!

✅ Approved. Can be merged as soon as the parent PR gets merged.

**Review by tjorim — DISMISSED** [review]
Looks good!

⚠️ The `ha_release` will need an update before merging this one in.

File: source/_components/notify.tplink_lte.markdown [hunk 1]
```diff
@@ -0,0 +1,17 @@
...
+title: "TP-Link LTE Notify"
+description: "Instructions on how to add TP-Link LTE notifications to Home Assistant."
+date: 2018-10-03 21:30
+sidebar: true
+comments: false
+sharing: true
+footer: true
+logo: tp-link.png
+ha_category: Notifications
+ha_release: "0.82"
```
Comment (MartinHjelmare) [inline source/_components/notify.tplink_lte.markdown]: 0.83

File: source/_components/tplink_lte.markdown [hunk 2]
```diff
@@ -0,0 +1,59 @@
...
+layout: page
+title: "TP-Link LTE"
+description: "Instructions on how to integrate your TP-Link LTE routers within Home Assistant."
+date: 2018-10-03 21:30
+sidebar: true
+comments: false
+sharing: true
+footer: true
+logo: tp-link.png
+ha_release: "0.82"
```
Comment (MartinHjelmare) [inline source/_components/tplink_lte.markdown]: 0.83

File: source/_components/notify.tplink_lte.markdown [hunk 3]
```diff
@@ -0,0 +1,17 @@
...
+---
```
Comment (MartinHjelmare) [inline source/_components/notify.tplink_lte.markdown:14]: Add `ha_iot_class`.

**Review by MartinHjelmare — DISMISSED** [review]

**Review by frenck — APPROVED** [review]

**Comment by frenck** [comment]
Thanks, @andtos90!

Corrected a small typo, but can be merged as soon as the build passes.

**Pull request merged**